simple_history/tests/tests/test_models.py

```diff
@@ -697,6 +698,18 @@ def test_history_diff_with_included_fields(self):
         self.assertEqual(delta.changed_fields, ["question"])
         self.assertEqual(len(delta.changes), 1)
 
+    def test_history_diff_with_non_editable_field(self):
+        p = PollWithNonEditableField.objects.create(
+            question="what's up?", pub_date=today
+        )
+        p.question = "what's up, man?"
+        p.save()
```

Comment:
Given the field `PollWithNonEditableField.created` is designated as `auto_now_add`, save() will not change it, so this test does not really exercise the exclusionary effect of `editable=False`.

Reply:
I'm happy to change this, and have now done so, but I would note the test was already testing the behaviour, because it doesn't matter if the save changes the field, the comparison of all fields is still performed.